lib/tasks/functional.yml

```diff
@@ -0,0 +1,93 @@
+---
+metadata:
+  name: '[TC] {{ platform|title }} firefox-ui-tests functional'
+  description: 'firefox-ui-tests functional run'
+  owner: 'mozauto@mozilla.com'
+  source: 'https://github.com/mozilla/mozmill-ci'
+
+created: '{{ now }}'
+deadline: '{{ fromNow("24 hours") }}'
+
+tags:
+  createdForUser: mozauto@mozilla.com
+
+provisionerId: 'aws-provisioner-v1'
+workerType: 'qa-3-linux-fx-tests'
+
+scopes:
+  - 'docker-worker:cache:level-3-{{ branch }}-test-workspace'
+
+routes:
+  - 'tc-treeherder{%if treeherder_instance is equalto "staging" %}-stage{% endif %}.{{ branch }}.{{ revision }}'
```

Comment:
Compared to in-tree tests which are only getting run in a single instance of taskcluster, we have a staging and production system too. That is the reason why we only send the report to a single Treeherder instance.